### PR TITLE
Changed automatic merge frequency to 'daily'

### LINKED ISSUE #813: Increase frequency of main website updates
## Description

Currently the main website gets updated from `development` every week:

https://github.com/ACCESS-NRI/access-hive.org.au/blob/17b5824bb2788dea39a42e18ca512c2a2cec7849/.github/workflows/automatic_merge.yml#L5-L7

This might be too infrequent.

## Solution
Change the scheduling of the merge from every week to every day (or every other day)

### PR DESCRIPTION
**Pull request by atteggiani**
Fixes #813.

### CODE REVIEW

File: .github/workflows/automatic_merge.yml [hunk 1]
```diff
@@ -2,9 +2,9 @@ name: Automatic merge
 
 on:
   schedule: 
-    # Run every Monday at 03:09 (AEDT)
-    # AEDT is UTC+10 --> Monday at 03:09 AEDT is Sunday at 17:09 UTC
-    - cron: "9 17 * * SUN"
+    # Run every day at 02:09 (AEDT)
```
Comment (harshula) [inline .github/workflows/automatic_merge.yml:5]:
```suggestion
    # Run every day at 01:09 (AEST)
```

File: .github/workflows/automatic_merge.yml [hunk 2]
```diff
@@ -2,9 +2,9 @@ name: Automatic merge
...
+    # AEDT is UTC+10 --> 02:09 AEDT is 16:09 UTC
```
Comment (harshula) [inline .github/workflows/automatic_merge.yml:6]:
```suggestion
    # AEST is UTC+10 --> 01:09 AEST is 14:09 UTC
```

Comment (harshula) [inline .github/workflows/automatic_merge.yml:6]:
```suggestion
    # AEST is UTC+10 --> 01:09 AEST is 15:09 UTC
```

File: .github/workflows/automatic_merge.yml [hunk 3]
```diff
@@ -2,9 +2,9 @@ name: Automatic merge
...
+    - cron: "9 16 * * *"
```
Comment (harshula) [inline .github/workflows/automatic_merge.yml:7]:
```suggestion
    - cron: "9 15 * * *"
```

File: .github/workflows/automatic_merge.yml (hunk 1 above)
Comment (atteggiani) [reply .github/workflows/automatic_merge.yml]: Addressed in [382c8b76bf08ead564929bb6220e3144def79b56](https://github.com/ACCESS-NRI/access-hive.org.au/pull/822/commits/382c8b76bf08ead564929bb6220e3144def79b56)

File: .github/workflows/automatic_merge.yml (hunk 2 above)
Comment (atteggiani) [reply .github/workflows/automatic_merge.yml]: Addressed in [382c8b76bf08ead564929bb6220e3144def79b56](https://github.com/ACCESS-NRI/access-hive.org.au/pull/822/commits/382c8b76bf08ead564929bb6220e3144def79b56)

File: .github/workflows/automatic_merge.yml (hunk 3 above)
Comment (atteggiani) [reply .github/workflows/automatic_merge.yml]: Addressed in [382c8b76bf08ead564929bb6220e3144def79b56](https://github.com/ACCESS-NRI/access-hive.org.au/pull/822/commits/382c8b76bf08ead564929bb6220e3144def79b56)

**Review by harshula — APPROVED** [review]

**Comment by atteggiani** [comment]
Thanks @harshula! 
Merged.

**Pull request merged**